core/src/main/resources/lib/form/checkbox.jelly

```diff
@@ -67,17 +67,19 @@ THE SOFTWARE.
   <j:set var="default" value="${attrs.default ?: false}"/>
   <j:set var="value" value="${attrs.checked ?: instance[attrs.field] ?: default}"/>
 
-  <input type="checkbox"
-         name="${name}"
-         value="${attrs.value}"
-         title="${attrs.tooltip}"
-         onclick="${attrs.readonly=='true' ? 'return false;' : attrs.onclick}" id="${attrs.id}" class="${attrs.class} ${attrs.negative!=null ? 'negative' : null} ${attrs.checkUrl!=null?'validated':''}"
-         checkUrl="${attrs.checkUrl}" checkDependsOn="${attrs.checkDependsOn}" json="${attrs.json}"
-         disabled="${readOnlyMode ? 'true' : null}"
-         checked="${value ? 'true' : null}"/>
-  <label class="attach-previous setting-checkbox ${attrs.title == null ? 'js-checkbox-label-empty' : ''}"
-         title="${attrs.tooltip}">${attrs.title}</label>
+  <span class="jenkins-checkbox">
```

Comment:
🚫  blocker: there's no focus state for checkboxes.

Tab over one and nothing happens, e.g. on the add credentials page

![image](https://user-images.githubusercontent.com/21194782/142162337-5873ae64-3641-4220-bd99-591bbc619949.png)

Reply:
<img width="389" alt="image" src="https://user-images.githubusercontent.com/43062514/143932687-fa96fdb1-b99d-4c96-972f-20501b5eecc7.png">

Fixed :)